<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomListItem</class>
 <widget class="QWidget" name="RoomListItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>215</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lbType">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>类型</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>主播名</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>标题</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbBorder">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>215</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid red</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPic">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>191</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbRoomMemNum">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>人数:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbPraiseNum">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>点赞:</string>
   </property>
  </widget>
  <zorder>lbBorder</zorder>
  <zorder>lbTitle</zorder>
  <zorder>lbType</zorder>
  <zorder>lbName</zorder>
  <zorder>btnPic</zorder>
  <zorder>lbRoomMemNum</zorder>
  <zorder>lbPraiseNum</zorder>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
